.prettierrc.yaml

```diff
@@ -0,0 +1,2 @@
+printWidth: 120
```

Comment:
FYI we changed here to 100 because ROS2's coding guideline is so.

Reply:
I'd like to fix it as you told.

Reply:
f8405941a353c96e23eeef2a2e3545b705ad3891